designs/administrator/4.5/quay-image-vulnerabilities/index.md

```diff
@@ -0,0 +1,22 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# Quay Container Security Operator Enhancements
+
+Enhancements have been made to the Container Security Operator, with the **Image Vulnerabilities** status on the cluster dashboard now having clearer labeling and access to the **Image Vulnerabilities** list. Also Pods now include a **Vulnerabilities** tab which lists out the vulnerable containers on the pod.
+
+## Image Vulnerabilities on Cluster Dashboard
+
+![Cluster Dashboard Image Vulnerabilities Status](img/imagevulndash45.png)
+- The **Quay Image Security** status has been renamed **Image Vulnerabilities** to better convey its purpose and relationship to the **Image Vulnerabilities** resources.
+- The popover now displays its data in a per image way, so the user knows just how many images are vulnerable on the cluster.
+- The **Fixable Container Images** section conveys how many namespaces are impacted by each image and how many of its vulnerabilities are fixable.
+-  There is now a **View all** link to immediately take the user to the **Image Vulnerabilities** list.
+
+## Image Vulnerabilities from Pods
+
+![Cluster Dashboard Image Vulnerabilities Status](img/podvulns.png)
+- Pods now include a **Vulnerabilities** tab that lists the containers on that pod that are vulnerable. This has replaced the **View Image Vulnerabilities** action.
```

Comment:
```suggestion
- The Pod Details view now include a **Vulnerabilities** tab that lists the containers on that pod that are vulnerable. This has replaced the **View Image Vulnerabilities** action.
```

Reply:
Does this exist even if there are no vulnerabilities?

Reply:
It does, not sure if it is something we can conditionally add?

Reply:
Yeah, my thought is that I wonder if people will assume this mean there are some vulnerabilities and click on it often to double check that there aren't? Likely they won't be at this level just browsing around though so...meh :) Maybe a small badge to show the number next to the tab name would be good to explore? A quick glance that it's 0 or more before I click on it.

Reply:
@itsptk Awesome! Thanks for pulling that together!